docs/source/useful_information/development/softwaredevelopment.rst

```diff
@@ -184,21 +186,53 @@ Then you will have to run::
 
     python -m pipenv install --dev
 
-
 Next, build the documentation in html format with the following commands run from the ``root`` folder::
 
     sphinx-apidoc -T -o docs/source/generated aequilibrae
     cd docs
     make html
 
+Working with progress bars
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+From version 1.1.0, AequilibraE is capable of displaying progress bars in Jupyter Notebooks using 
+`TQDM <https://tqdm.github.io/>`_. For the companion QGIS plugin, `PyQt5 <https://doc.qt.io/qtforpython-5/>`
+is used to emit messages in progress bars.
+
+In AequilibraE, there are specific pieces of code responsible for emitting these signals to progress bars
```

Comment:
This doesn't make sense to me - i still don't know what a signal emission is meant to look like or achieve.